libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Settings/RuntimeSettings.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Microsoft.Bot.Builder.Integration.Runtime.Plugins;
+using Microsoft.Bot.Builder.Runtime.Plugins;
+
+namespace Microsoft.Bot.Builder.Integration.Runtime.Settings
+{
+    /// <summary>
+    /// Settings for the bot runtime.
+    /// </summary>
+    internal class RuntimeSettings
```

Comment:
great that these are internal - but then given its internal and presumably used in exactly one place do we really need all this code?

Reply:
What's the proposed alternative? using strings to reference config sections?

Reply:
perhaps - there just seems like a lot of property bag classes

Reply:
Let's brainstorm on this next week after R12.  I agree that this needs some work.